pkg/event/target/mqtt.go

```diff
@@ -135,13 +144,17 @@ func NewMQTTTarget(id string, args MQTTArgs) (*MQTTTarget, error) {
 		AddBroker(args.Broker.String())
 
 	if args.QueueDir != "" {
-		options = options.SetStore(mqtt.NewFileStore(args.QueueDir))
+		options = options.SetStore(mqtt.NewFileStore(args.QueueDir)).SetIgnoreAndRetryConnection(true).SetMaxReconnectInterval(retryTimeout * time.Second)
```

Comment:
Move `5 * time.Second` as default inside `paho.mqtt.golang` and this option will be avoided here.

Reply:
`SetConnectionRetry()` is better than `SetIgnoreAndRetryConnection()`

Reply:
Done :+1: